research/src/pages/popup/popover.research.explainer.mdx

```diff
@@ -625,8 +605,11 @@ Note that using the API described in this explainer, it is possible for elements
 
 ```html
 <my-popover>
-  <template shadowroot=closed>
-    <div popover=auto>This is a popover: <slot></slot></div>
+  <template shadowroot="closed">
```

Comment:
Given recent declarative shadow dom conversations, sounds like we should make this `<template shadowrootmode="closed">`. But again, for another day.